centreon/www/front_src/src/AgentConfiguration/Form/ConnectionInitiated/HostConfigurations/HostConfiguration.cypress.spec.tsx

```diff
@@ -81,7 +81,7 @@ describe('HostConfiguration', () => {
   });
 
   it('shows certificate fields when connection mode is secure', () => {
-    cy.get('[data-testid="CA (.crt,.cer)"]').should('be.visible');
+    cy.get('[data-testid="CA (.cert,.cer)"]').should('be.visible');
```

Comment:
```suggestion
    cy.get('[data-testid="CA (.crt,.cert,.cer)"]').should('be.visible');
```
I guess you are supporting an additional format, not replacing the .crt one. To ask to @fmattesct

Reply:
Exactly